data/brands/amenity/cafe.json

```diff
@@ -2372,6 +2372,47 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Pellini",
+      "id": "pellini-9278bc",
+      "locationSet": {
+        "include": [
+          "tr"
+        ]
+      },
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Pellini",
+        "brand:wikidata": "Q119560268",
+        "cuisine": "coffee_shop",
+        "name": "Pellini",
+        "takeaway": "yes"
+      }
+    },
+    {
+      "displayName": "Shakespeare Coffee & Bistro",
+      "id": "shakespeare-9278bc",
+      "locationSet": {
+        "include": [
+          "tr", "az"
+        ]
+      },
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Shakespeare Coffee & Bistro",
+        "brand:wikidata": "Q119561507",
+        "cuisine": "bistro;coffee_shop;turkish;breakfast;beef;chicken;salad;pasta;pizza;dessert",
```

Comment:
NSI isn't a menu, can you shorten the cuisine list and only focus on major one (or two)?

Also for `drink:*` keys.

Reply:
Ok, removed.